guides/use-microprofile-amqp-connector-with-ssl-openshift.adoc

```diff
@@ -0,0 +1,194 @@
+
+[NOTE]
+--
+You must be logged by `oc` command for broker certificate to have correct domain of Openshift cluster for hostname. This is necessary for hostname verification during TLS handshake where client is verifying brokers hostname in the certificate.
+--
+
+[source, bash ,options="nowrap"]
+----
+# Generate broker's key pair using obtained OpenShift cluster domain for `apps`
+keytool -genkeypair -alias broker -keyalg RSA -storetype PKCS12 -keystore broker.ks -storepass changeit -validity 365 -dname "CN=*.apps.`oc whoami --show-server=true | sed 's~https://api\.~~;s~:[0-9]*~~'`, OU=My Org Unit, O=My Organization, L=My City, S=My State, C=My Country"
+# Generate key pair for reactive mesasaging application
+keytool -genkeypair -alias client -keyalg RSA -storetype PKCS12 -keystore client.ks -storepass changeit -validity 365 -dname "CN=localhost, OU=My Org Unit, O=My Organization, L=My City, S=My State, C=My Country"
+# Export broker certificate
+keytool -export -alias broker -file broker.cert -keystore broker.ks -storepass changeit
+# Export client certificate
+keytool -export -alias client -file client.cert -keystore client.ks -storepass changeit
+# Import broker and client certificate into a single truststore for simplicity
+keytool -import -v -trustcacerts -alias client -file client.cert -keystore client.ts -storepass changeit -noprompt
+keytool -import -v -trustcacerts -alias broker -file broker.cert -keystore client.ts -storepass changeit -noprompt
+----
+
+=== Deploy AMQ 7 broker
+First we need to create secret with server keystore and truststore which will be used by AMQ 7 broker:
+[source, bash]
+----
+oc create secret generic brokeramqp-ssl-secret --from-file=broker.ks --from-file=client.ts --from-literal=keyStorePassword='changeit' --from-literal=trustStorePassword='changeit'
+----
+
+Then create file `broker.yaml` with following content referencing above secret:
+[source, yaml]
+----
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: amq-broker
+  application: amq-broker-app
+spec:
+  acceptors:
+    - port: 61617
+      verifyHost: false
+      needClientAuth: true
+      expose: true
+      multicastPrefix: jms.topic.
+      name: all
+      connectionsAllowed: 10
+      sslEnabled: true
+      protocols: all
+      sslSecret: brokeramqp-ssl-secret
+      sslProvider: JDK
+      anycastPrefix: jms.queue.
+  console:
+    expose: true
+  deploymentPlan:
+    journalType: nio
+    messageMigration: false
+    persistenceEnabled: false
+    requireLogin: false
+    size: 1
+  upgrades:
+    enabled: false
+    minor: false
+----
+
+To simplify this example, there is set `verifyHost: false` to avoid need to set correct hostname in the "client" certificate for reactive messaging application.
+
+Run following command to deploy AMQ 7 broker with secured acceptor:
+[source, shell]
+----
+oc create -f broker.yaml
```

Comment:
is there a step here to check that the broker is properly deployed and up & running?

Reply:
Adding it.